models/tmp/stg_pinterest_ads__ad_group_history_tmp.sql

```diff
@@ -1,4 +1,14 @@
 {{ config(enabled=var('ad_reporting__pinterest_ads_enabled', True)) }}
 
-select *
-from {{ var('ad_group_history') }}
+{{
+    fivetran_utils.union_data(
+        table_identifier='ad_group_history', 
+        database_variable='pinterest_database', 
+        schema_variable='pinterest_schema', 
+        default_database=target.database,
+        default_schema='pinterest_ads',
```

Comment:
Same comment from LinkedIn. Can we make any necessary updates here to ensure we do not need to update fivetran_utils.

Reply:
Same comment for other tmp models.

Reply:
If we only needed to update the identifiers, was there a need to update the schema and database variables?

Reply:
Yes! I did update the schema and database variables. By "only identifiers" I meant in terms of what was changed from main and in comparison to linkedin.

Reply:
I was thinking to standardize all the names, but since it is a big change, I see what you mean. I have put the schema and database variables back.